src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Broker/IBroker.cs

```diff
@@ -23,26 +27,16 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 //
-//------------------------------------------------------------------------------
-
-using Microsoft.Identity.Core;
-using Microsoft.Identity.Core.Cache;
-using System.Collections.Generic;
-using System.Threading.Tasks;
+//
 
-namespace Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Platform
+namespace Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Broker
 {
-    internal class BrokerHelper
```

Comment:
All the broker changes should be pretty isolated. I needed to them because it was impossible to inject a logger into the BrokerProxy and also because it will make the code much easier to understand.